doc/DeveloperGuide.md

```diff
@@ -77,6 +77,17 @@ Priority | As a ... | I want to ... | So that I can...
 4. AddressBook deletes the person <br>
 Use case ends.
 
+#### Use case: Rename of an existing tag
```

Comment:
You added this in the middle of the previous use case. Can you think of any extensions your use case should have?